tsfc/driver.py

```diff
@@ -361,6 +362,13 @@ def compile_expression_dual_evaluation(expression, to_element, coordinates, inte
         point_set = PointSet(qpoints)
         config = kernel_cfg.copy()
         config.update(point_set=point_set)
+
+        # Allow interpolation onto QuadratureElements to refer to the quadrature
+        # rule they represent
+        if isinstance(to_element, FIAT.QuadratureElement):
+            quad_rule = QuadratureRule(point_set, to_element._weights)
```

Comment:
I know it's trivially true, but can you `assert` that the `point_set` points we've collected (`qpoints`) match the `to_element._points`?

Reply:
Done.